daemon/delete.go

```diff
@@ -26,7 +26,7 @@ func (daemon *Daemon) ContainerRm(name string, config *types.ContainerRmConfig)
 
 	// Container state RemovalInProgress should be used to avoid races.
 	if inProgress := container.SetRemovalInProgress(); inProgress {
-		return nil
+		return fmt.Errorf("container %s removal is in progress by other client", name)
```

Comment:
@cpuguy83 Oh, I think the "removal of ..." is better. I will update this PR. Thanks.